src/AnalyticsEventReporter.ts

```diff
@@ -66,6 +83,20 @@ export class AnalyticsEventReporter implements AnalyticsEventService {
     const shouldUseBeacon = useBeacon(finalPayload, this.config.forceFetch);
     const requestUrl = setupRequestUrl(this.config.env, this.config.region);
 
+    if (this.config.debug) {
+      console.log(
+        '[DEBUG] The following Payload would be sent to the Yext Events API using ' +
+          (shouldUseBeacon ? 'Beacon. ' : 'fetch(). ') +
+          'Session Tracking is ' +
+          (this.config.sessionTrackingEnabled ? 'enabled.\n' : 'disabled.\n') +
+          'Request URL: ' +
+          requestUrl +
+          '\n' +
+          'To send the event to the Yext Events API, disable the debug flag in your AnalyticsConfig.\n',
```

Comment:
This sentence seems unnecessary. Someone should know if they've enabled debug mode or not.